README.md

```diff
@@ -410,6 +410,39 @@ allow = response {
 }
 ```
 
+With denied response,
+array of headers can be passed to the downstream client.
```

Comment:
Can we add a short note on what the results are in the client?

Maybe something along the lines of
> With the following policy, a `POST /admin` request would result in the following response, featuring multiple `why` header values:
> ```
> 400 Bad Request
> why: POST method is not allowed
> why: /admin path is not allowed
>
> Unauthorized Request
> ```

Reply:
We should rephrase the text a little. Something like:

> The following policy shows how to generate an `object` response with the `headers` set as an array of values. The example policy restricts access to the `/admin` path and the `POST` method. 

<**add_ the_policy_example_here**>

The input to policy is show below:
```json
{
    	"attributes": {
        	"request": {
            	    "http": {
                	    "method": "POST",
                            "path": "/admin"
                    }
              }
        }
    }
```


The response generated by the policy for the above input looks like below:
```json
{
            "allowed": false,
            "body": "Unauthorized Request",
            "headers": [
                {
                    "why": "/admin path is not allowed"
                },
                {
                    "why": "POST method is not allowed"
                }
            ],
            "http_status": 400
}
```